deepchem/models/tests/test_jax_model.py

```diff
@@ -39,14 +40,15 @@ def rms_loss(pred, tar, w):
 
   # JaxModel Working
   j_m = JaxModel(
-      model,
+      model.apply,
       params,
       criterion,
       batch_size=256,
       learning_rate=0.001,
       log_frequency=2)
-  results = j_m.fit(dataset, deterministic=True)
-  assert results < 0.5
+  _ = j_m.fit(dataset, nb_epochs=25, deterministic=True)
```

Comment:
about how long does this training take?

Reply:
They are quite fast as the datasets are small and apart from that I'm running it on `jit` .